### PR TITLE
add example how to enable deployTektonTaskResources feature gate in hco cr

### PR DESCRIPTION
**Pull request by ksimon1**
add example how to enable deployTektonTaskResources feature gate in hco cr

tekton tasks are not deployed by default. This example shows user how to update feature gate to deploy tekton tasks.
Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: docs/virtual_machines/tekton_tasks.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ spec:
   featureGates:
     deployTektonTaskResources: true
 ```
+User can use this command to enable `deployTektonTaskResources` feature gate in HCO CR
+```console
+oc patch hco kubevirt-hyperconverged  --type=merge -p '{"spec":{"featureGates": {"deployTektonTaskResources": true}}}'
+```
+or in TTO CR
```
Comment (dominikholler) [inline docs/virtual_machines/tekton_tasks.md:51]: Does this apply only to stand-alone without HCO?

**Review by dominikholler — COMMENTED** [review]
Thanks for the PR!

File: docs/virtual_machines/tekton_tasks.md (hunk 1 above)
Comment (ksimon1) [reply docs/virtual_machines/tekton_tasks.md:51]: yes

Comment (dominikholler) [reply docs/virtual_machines/tekton_tasks.md:51]:
```suggestion
or in TTO CR if TTO is deployed as a stand-alone without HCO
```

**Review by dominikholler — APPROVED** [review]
/lgtm 
Thanks

**Comment by ksimon1** [comment]
@dhiller can you please review this PR?

**Comment by ksimon1** [comment]
@davidvossel can you please review?

**Comment by ksimon1** [comment]
@jobbler, @rmohr can you please review this PR?

**Comment by ksimon1** [comment]
@stu-gott please do you know who can merge this?

**Review by dhiller — COMMENTED** [review]
/approve

**Pull request merged**